adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -144,10 +189,10 @@ object ReferenceRegion {
  *            which is <i>not</i> in the region -- i.e. [start, end) define a 0-based
  *            half-open interval.
  */
-case class ReferenceRegion(referenceName: String, start: Long, end: Long) extends Ordered[ReferenceRegion] with Interval {
+case class ReferenceRegion(referenceName: String, start: Long, end: Long) extends Interval {
```

Comment:
Well, IMO, there's a bit more to the story than that. The big advantage of moving `ReferenceRegion` to Avro, is that we could clean up our Avro records. `AlignmentRecord` would not have start, end anymore. It would have a `ReferenceRegion`. Ditto for `Variant`, `Feature`, `NucleotideContigFragment`, etc.